### PR TITLE
added MMv1 and handwritten readme

### PR DESCRIPTION
**Pull request by shuyama1**
Rewrite of the magic module readme page will be in a separate PR. 
Given the docs fixit is coming next week and just want to make sure we can have this setup ready prior to the fixit week.

If this PR is for Terraform, I acknowledge that I have:

- [ ] Searched through the [issue tracker](https://github.com/hashicorp/terraform-provider-google/issues) for an open issue that this either resolves or contributes to, commented on it to claim it, and written "fixes {url}" or "part of {url}" in this PR description. If there were no relevant open issues, I opened one and commented that I would like to work on it (not necessary for very small changes).
- [ ] [Generated Terraform](https://github.com/GoogleCloudPlatform/magic-modules#generating-the-terraform-providers), and ran `make test` and `make lint` to ensure it passes unit and linter tests.
- [ ] Ensured that all new fields I added that can be set by a user appear in at least one [example](https://github.com/GoogleCloudPlatform/magic-modules/tree/main/mmv1/templates/terraform/examples) (for generated resources) or [third_party test](https://github.com/GoogleCloudPlatform/magic-modules/tree/main/mmv1/third_party/terraform/tests) (for handwritten resources or update tests).
- [ ] [Ran](https://github.com/hashicorp/terraform-provider-google/blob/main/.github/CONTRIBUTING.md#tests) relevant acceptance tests (If the acceptance tests do not yet pass or you are unable to run them, please let your reviewer know).
- [ ] Read the [Release Notes Guide](https://github.com/GoogleCloudPlatform/magic-modules/blob/main/.ci/RELEASE_NOTES_GUIDE.md) before writing my release note below.

**Release Note Template for Downstream PRs (will be copied)**

```release-note:none

```

### CODE REVIEW

File: mmv1/third_party/README.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
+# Handwritten
+
+## Overview
+
+The Google providers for Terraform have a large number of handwritten go files, written before Magic Modules was used with them. While conversion is ongoing, many resources are still managed by hand.
```
Comment (rileykarson) [inline mmv1/third_party/README.md]:
```suggestion
The Google providers for Terraform have a large number of handwritten go files, primarily for resources written before Magic Modules was used with them. Most handwritten files are expected to stay handwritten indefinitely, although conversion to a generator may be possible for a limited subset of them.
```

We haven't actively done conversions in a while, so let's message slightly more strongly against doing so.

File: mmv1/README.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+	- [Test](#test)
+	- [Documentation](#documentation)
+	- [Beta Feature](#beta-feature)
+		- [Add or update a beta future](#add-or-update-a-beta-feature)
+		- [Test that use a beta feature](#test-that-use-a-beta-feature)
+		- [Promote a beta feature](#promote-a-beta-feature)
+
+## Contributing
+
+We'd love to accept your contributions! Thanks for making the changes :) Here's some guidance on how to contribute to mmv1-genereated resources.
```
Comment (rileykarson) [inline mmv1/README.md:22]:
```suggestion
We're glad to accept contributions to {{type}} resources. Tutorials and guidance on making changes are available below.
```

Maybe a slightly more clinical wording, i.e. the suggestion, subbing type? So `MMv1` here, and `handwritten` below. By contrast, in tpgtools, we'd state we don't accept contributions, I'd assume.

File: mmv1/README.md [hunk 3]
```diff
@@ -0,0 +1,38 @@
+# MMV1
+
+## Overview
+
+Mmv1 is a code generator that implements the Terraform Google Provider (TGP) resources from ruby scripts.
```
Comment (rileykarson) [inline mmv1/README.md:5]:
```suggestion
Mmv1 is a code generator that implements Terraform Provider Google (TPG) resources from ruby scripts.
```

nit: TPG > TGP generally, named for the https://github.com/hashicorp/terraform-provider-google repo. Terraform Google Provider sounds better, admittedly, I typically call it "the `google` provider" when not using TPG.

Comment (rileykarson) [inline mmv1/README.md:5]: WDYT of something like "MMv1 is a Ruby-based code generator that ... from YAML specification files. Contributions typically require editing YAML files and small snippets of Go code."? Other than by folks on our team, Ruby contributions are pretty rare.

Comment (shuyama1) [reply mmv1/README.md:5]: Totally make sense! I'll change it to "MMv1 is a Ruby-based code generator that implements Terraform Provider Google (TPG) resources from YAML specification files." or should I also mention TPGB? But I was thinking TPG may be a general term for the google providers.

Comment (rileykarson) [reply mmv1/README.md:5]: Yeah- TPG generally means both.

File: mmv1/README.md [hunk 4]
```diff
@@ -0,0 +1,38 @@
+# MMv1
+
+## Overview
+
+MMv1 is a Ruby-based code generator that implements Terraform Provider Google (TPG) resources from YAML specification files.
+
+MMv1-generated resources like [google_compute_address](https://registry.terraform.io/providers/hashicorp/google/latest/docs/resources/compute_address) can be identified by looking in their [Go source](https://github.com/hashicorp/terraform-provider-google/blob/main/google/resource_compute_address.go) for an `AUTO GENERATED CODE` header as well as a Type `MMv1`. MMv1-generated resources should have source code present under their product folders, like [mmv1/products/compute](./products/compute) for [google_compute_address](https://registry.terraform.io/providers/hashicorp/google/latest/docs/resources/compute_address) resource.
```
Comment (rileykarson) [inline mmv1/README.md:7]:
```suggestion
MMv1-generated resources like [google_compute_address](https://registry.terraform.io/providers/hashicorp/google/latest/docs/resources/compute_address) can be identified by looking in their [Go source](https://github.com/hashicorp/terraform-provider-google/blob/main/google/resource_compute_address.go) for an `AUTO GENERATED CODE` header as well as a Type `MMv1`. MMv1-generated resources should have source code present under their product folders, like [mmv1/products/compute](./products/compute) for the [google_compute_address](https://registry.terraform.io/providers/hashicorp/google/latest/docs/resources/compute_address) resource.
```

**Review by rileykarson — APPROVED** [review]
Frustrating!

**Pull request merged**